site/pages/category.json

```
@@ -0,0 +1,14 @@
+{
+    "route": "shop/:name/:id",
+    "template": "templates/store/category",
+    "props": {
+        "category": {
+            "filters": {
+                "id": "request_data(path.id)"
```

We'll be requiring `square_online_id:true` soon
```suggestion
                "id": "request_data(path.id)",
                "square_online_id": true
```